meta/main.yml

```
@@ -20,4 +20,4 @@ galaxy_info:
     - logmanagement
     - elasticstack
 
-dependencies: []
+dependencies: []
```

```suggestion
dependencies: []
```
This is listed as "replaced". Maybe there's some unprintable character?

---

Seems like that was just a display bug in git / GitHub